### PR TITLE
 #167727723 deploy app to heroku using docker

### PR DESCRIPTION
**Pull request by tobslob**
#### What does this PR do?
This PR deploy app to heroku using docker
#### Description
- add services to .travis.yml
- configure before_install in .travis.yml
- install dependencies
- run script on travis
- deploy to docker
- push to heroku

#### How can this be manually tested
- access app through ``` https://storm-backend.herokuapp.com ```

[maintains #167727723 ]

### CODE REVIEW

File: .env-sample [hunk 1]
```diff
@@ -14,3 +14,10 @@ APP_EMAIL = ""
 SEND_GRID_API_KEY= ""
 BASE_URL = "http://localhost:3000"
 SIGNIN_PAGE = ""
+
+DOCKER_PASSWORD=""
+DOCKER_USERNAME=""
+HEROKU_PASSWORD=""
+HEROKU_USERNAME=""
```
Comment (tejiri4) [inline .env-sample]: do we need all these? are they needed

File: heroku.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+build:
+  docker:
+    web: Dockerfile
+run:
+  web: docker-compose up
+setup:
+  addons:
+  - plan: heroku-postgresql
+    as: DATABASE
```
Comment (tejiri4) [inline heroku.yml]: add new line here

File: .env-sample (hunk 1 above)
Comment (tobslob) [reply .env-sample]: Not anymore, I have removed it

File: heroku.yml (hunk 2 above)
Comment (tobslob) [reply heroku.yml]: Okay, done

**Review by tejiri4 — APPROVED** [review]
LGTM

**Pull request merged**